www/scripts/generate-cli-docs.js

```diff
@@ -21,8 +21,10 @@ function runCommand(command, args) {
 }
 
 function parseOutput(command, output) {
+  const sectionToken = '\^/--IMATOKEN--\^/';
+
   const lines = output.split('\n');
-  const sections = output.split('\n\n');
+  const sections = output.replace(/\n\n(.+):\n/g, sectionToken + '$1:').split(sectionToken);
```

Comment:
Do we need to do this dance with a custom `sectionToken` if we already know enough to place that custom token correctly in the first place?

Reply:
The regex matches on two empty spaces followed by some text and then a colon. The text is the section heading. I can split on that and along with the capture group for the heading text to get an array with the heading text followed by the section text, but then I would have to refactor how we break out and markup sections later on. I went with a token because it seemed like a simpler fix in this instance and I think the pattern leads to a bit more flexibility in the future, namely with fixing some of the spacing issues that we still have.